### ISSUE TITLE
Introducing a fork 

### ISSUE THREAD

**Issue by vldm** [stoically/syn-rsx#58]
Hi! I have found that @stoically didn't contribute to a project for a while.

I have mailed him, but no answer.
Maybe some one has dirrect contact to him and can ask, if everything ok, or not.

But for now, while no answer is received. I have createad a fork https://github.com/rs-tml/rstml

This fork has few major changes:
- Massive refactor to types
- Now types has it lossless form (every parsed token is saved in Node) so every type can roundtrip parse/to_tokens.
- Unquoted text implemented
- Better IDE support

Is there any interest in it?

link to comparsion: https://github.com/rs-tml/rstml/blob/main/comparsion-with-syn-rsx.md

**Comment by erlend-sh** [comment]
Seems like a necessary path forward so long as syn-rsx doesn’t have an active maintainer and no way of doing a hand-off.

I think both Dioxus and Leptos devs will be supportive of this.

Hopefully all is well with @stoically 🙏 since syn-rsx is completely halted, it wouldn’t be hard to join the two project together in the future if there’s mutual interest in that.

**Comment by vldm** [comment]
> it wouldn’t be hard to join the two project together in the future if there’s mutual interest in that.

Yes, thats why i have renamed the fork, currently it already contain breaking change because of new "lossless form".
But I'm sure if the need arises, it will still be possible to merge the work and return to the original fork.

I really like the idea of @stoically - to make generic library with "rsx" syntax, for all web-related projects.
And don't want this project to stop.

I also have created a organization, in hope, that some old contributors of syn-rsx can became mainteners.

**Comment by gauteh** [comment]
https://github.com/stoically/temporary-containers/issues/618
